app/react/stories/dragAndDrop/DragAndDrop.stories.tsx

```diff
@@ -0,0 +1,255 @@
+/* eslint-disable react/no-multi-comp */
```

Comment:
I would move the story outside the folder, and maybe just use this folder for the components used in the story to simplify the file?